pygen/pygen_src/isa/riscv_instr.py

```diff
@@ -73,117 +85,156 @@ def __init__(self):
         self.label = ""
         self.is_local_numeric_label = None
         self.idx = -1
-        self.has_rs1 = 1
-        self.has_rs2 = 1
-        self.has_rd = 1
-        self.has_imm = 1
+        self.has_rs1 = vsc.uint32_t(1)
+        self.has_rs2 = vsc.uint32_t(1)
+        self.has_rd = vsc.uint32_t(1)
+        self.has_imm = vsc.uint32_t(1)
```

Comment:
I think since these are all "single bit" variables, it makes more sense to use `vsc.bit_t` type rather than `vsc.uin32_t(1)`.